client/my-sites/site-settings/spam-filtering-settings.jsx

```diff
@@ -0,0 +1,129 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { includes } from 'lodash';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import FoldableCard from 'components/foldable-card';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormTextInput from 'components/forms/form-text-input';
+import FormInputValidation from 'components/forms/form-input-validation';
+import InfoPopover from 'components/info-popover';
+import ExternalLink from 'components/external-link';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getJetpackSettingsSaveError, getJetpackSettingsSaveRequestStatus } from 'state/selectors';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+
+const SpamFilteringSettings = ( {
+	fields,
+	dirtyFields,
+	hasAkismetKeyError,
+	isRequestingSettings,
+	isSavingSettings,
+	onChangeField,
+	translate
+} ) => {
+	const {
+		wordpress_api_key,
+		akismet: akismetActive
+	} = fields;
+	const isDirty = includes( dirtyFields, 'wordpress_api_key' );
+	const header = ( akismetActive && wordpress_api_key && ! isDirty )
+		? translate( 'Your site is protected from spam.' )
+		: translate( 'Your site needs an Antispam key.' );
+
+	const inTransition = isRequestingSettings || isSavingSettings;
+	const validationText = hasAkismetKeyError
+		? translate( 'There\'s a problem with your Antispam API key.' )
+		: translate( 'Your Antispam key is valid.' );
+	const className = ( inTransition || isDirty )
+		? null
+		: classNames( {
+			'is-valid': wordpress_api_key && ! hasAkismetKeyError,
+			'is-error': wordpress_api_key && hasAkismetKeyError
+		} );
+	return (
+		<div className="site-settings__spam-filtering">
+			<FoldableCard header={ header }>
+				<FormFieldset>
+					<div className="spam-filtering-settings__info-link-container site-settings__info-link-container">
+						<InfoPopover >
+							<ExternalLink
+								target="_blank"
+								icon
+								href={ 'https://jetpack.com/features/security/spam-filtering/' }
+							>
+								{ translate( 'Learn more about spam filtering.' ) }
+							</ExternalLink>
+						</InfoPopover>
+					</div>
+					<div className="site-settings__child-settings">
+						<FormLabel htmlFor="wordpress_api_key">{ translate( 'Your API Key ' ) }</FormLabel>
```

Comment:
Hi! I've found a possible matching string that has already been translated **42** times:
`translate( 'API Key' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/akismet-card-for-jetpack-sites-in-settings-page)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).